test/clj_sqs_extended/test_fixtures.clj

```diff
@@ -0,0 +1,54 @@
+(ns clj-sqs-extended.test-fixtures
+  "Provides some helper functions to provide a convenient testing environment
```

Comment:
this is obvious enough from the namespace name (good for you), so I'd remove this docstring since it doesn't add anything useful